vllm/engine/arg_utils.py

```diff
@@ -784,6 +784,11 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             },
             default=None,
             help="override or set neuron device configuration.")
+        parser.add_argument(
+            "--disable-fastapi-docs",
+            action='store_true',
+            default=False,
+            help="Disable OpenAPI schema, Swagger UI, and ReDoc documentation")
```

Comment:
change this to the openai api server's args please

Reply:
sounds good. Let me make the change